ocs_ci/utility/utils.py

```diff
@@ -905,6 +909,41 @@ def get_cluster_version_info():
     return cluster_version_info
 
 
+def get_ocs_operator_version():
+    """
+    Gets the ocs operator version info
+
+    Returns:
+        str: ocs operator version
+
+    """
+    from ocs_ci.ocs.ocp import OCP
+    ocp = OCP(kind="catalogsource", namespace="openshift-marketplace")
+    for item in ocp.get().get('items'):
+        if item.get('metadata').get('name') == "ocs-catalogsource":
```

Comment:
check: ocs_ci/ocs/constants.py and OPERATOR_CATALOG_SOURCE_NAME

Reply:
Done